sdlib/d/gc/bin.d

```diff
@@ -55,21 +55,13 @@ struct Bin {
 		assert(&arena.bins[pd.sizeClass] == &this,
 		       "Invalid arena or sizeClass!");
 
-		auto e = pd.extent;
-		auto sc = pd.sizeClass;
-
-		import d.gc.util;
-		auto offset = alignDownOffset(ptr, PageSize) + pd.index * PageSize;
-		auto index = binInfos[sc].computeIndex(offset);
-		auto slots = binInfos[sc].slots;
-
-		auto base = ptr - offset;
-		assert(ptr is base + index * binInfos[sc].itemSize);
```

Comment:
This assert is gone now.

Reply:
I don't think it belongs in slab, because it's not inherently wrong to not pass the base pointer in there.

Reply:
But i see you put it this function, so the code is correct.